.github/workflows/backfill-corr-ci.yml

```diff
@@ -10,57 +10,37 @@ name: R backfill corrections
 
 on:
   push:
-    branches: [ main, prod ]
+    branches: [main, prod]
   pull_request:
-    types: [ opened, synchronize, reopened, ready_for_review ]
-    branches: [ main, prod ]
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches: [main, prod]
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     if: github.event.pull_request.draft == false
-    strategy:
-      matrix:
-        r-version: [4.2.1]
```

Comment:
suggestion (optional): maybe we want to keep the `matrix` way of providing `r-version` for future flexibility? In some other packages, we specify several different R versions to run tests on.